### ISSUE TITLE
[20220619] Weekly AI ArXiv 만담 - 55회차

### ISSUE THREAD

**Issue by jungwoo-ha** [jungwoo-ha/WeeklyArxivTalk#55]
### News
- Conferences
  - Interspeech 2022: Notification: 모두 축하드립니다. 송도에서 뵈요!
  - CVPR 2022: 오늘부터 드디어 시작!!! 부스 많이 들러 주세요! (6.19 ~ 24)
    - 네이버의 발표 스케쥴 (17개 발표): https://naver-career.gitbook.io/en/teams/clova-cic/events/clova-ai-lab-cvpr-2022
  - ACM FAccT 2022: 6.21~24: https://facctconference.org/2022/ 
    - HyperscaleFAccT CRAFT (6.21): https://naver-career.gitbook.io/en/teams/clova-cic/events/hyperscalefacct-facct-2022
    - Tutorial (6.22): [Shortcut Learning in Machine Learning: Challenges, Analysis, Solutions (In-person)](https://facctconference.org/2022/acceptedtuts.html#short) 
- [HyperCLOVA 연구용으로 드디어 일반 공개 (선신청 노고민)](https://www.nipa.kr/main/selectBbsNttView.do?key=122&bbsNo=2&nttNo=2259&bbsTy=biz&searchCtgry=&searchCnd=all&searchKrwd=&pageIndex=1)
  - 대상: 공공기관, 연구기관, 대학교(원)

### ArXiv
- [Disentangling visual and written concepts in CLIP](https://arxiv.org/abs/2206.07835)
  - CLIP에서 written text 와 visual concept이 entangle 되어 있는데 이걸 distentangle이 가능한지 연구(from MIT)
  - Learn to spell 과 Fortget to spell 이 되도록 각각 학습해서 disentangle 시켜봄
  - 각각의 feature로 text-control 이미지 생성결과 및 image2text retrieval 성능 비교
![image](https://user-images.githubusercontent.com/11782739/174467350-8b91d894-cebb-4a48-9aa6-b35aa9c27ef2.png)
![image](https://user-images.githubusercontent.com/11782739/174467376-3a627395-b7c2-45aa-8e9e-03deabcb7514.png)
![image](https://user-images.githubusercontent.com/11782739/174467398-d11a2ba7-0c30-428f-b323-7d460a2941ab.png)

- [OmniMAE: Single Model Masked Pretraining on Images and Videos](https://arxiv.org/abs/2206.08356)
  - Meta AI [Omnivore-CVPR 2022 oral](https://arxiv.org/abs/2201.08377) 에 MAE로 Image와 Video Joint Self-supervised learning 연구 (MAE + VideoMAE)
  - 특별한 모델구조 변경없이 spatial (+temporal) patch -> 다수 masking -> encoder -> decoder -> pixel recon 하는 MAE구조
  - Joint 트레이닝하니 개별 학습보다 좋은 성능을 보인다고. 
  - Pretraining data는 ImageNet-1k와 SSv2, Downstream은 iNaturalist-2018, Place365, K400, Epic-Kitchen등
  - Masking 관련 ablation은 다른 비디오나 이미지 연구에도 도움될 듯
![image](https://user-images.githubusercontent.com/11782739/174461314-5e887514-070b-4e4c-a4f2-f5f9b56ff199.png)
![image](https://user-images.githubusercontent.com/11782739/174461333-9ac4d75a-11c3-4f09-bd5b-8c90e7542f97.png)
![image](https://user-images.githubusercontent.com/11782739/174461382-5c0dd603-600d-4075-98c2-d474d9913b2c.png)

- [Beyond Supervised vs. Unsupervised: Representative Benchmarking and Analysis of Image Representation Learning](https://arxiv.org/abs/2206.08347)
  - 다양한 SSL method들이 나오는데 성능을 제대로 평가하기 위한 비교방법고 두개의 새로운 평가지표 제안 (CVPR 2022)
  - https://mgwillia.github.io/exploring-unsupervised/

- [BYOL-Explore: Exploration by Bootstrapped Prediction](https://arxiv.org/abs/2206.08332)
  - RL하시는 분들은 참조해보세요~~

- [Characteristics of Harmful Text: Towards Rigorous Benchmarking of Language Models](https://arxiv.org/abs/2206.08325)
  - 언어모델 연구 + 서비스하시는 분들을 위하여.

- [Language Models are General-Purpose Interfaces](https://arxiv.org/abs/2206.06336)
  - MSR에서 나온 Image + Text + Multiligual 동시 학습 LM (Semi-causal LM이라고)
  - https://github.com/microsoft/unilm 에서 다운 가능 (아직은 껍데기만)

**Comment by music-ai-644** [comment]
### Arxiv (Audio and Speech Processing)

- [Automatic Prosody Annotation with Pre-Trained Text-Speech Model](https://arxiv.org/pdf/2206.07956.pdf)
- [GoodBye WaveNet - A Language Model for Raw Audio with Context of 1/2 Million Samples](https://arxiv.org/pdf/2206.08297.pdf)
- 흥미 있는 연구
    - [To Dereverb Or Not to Dereverb? Perceptual Studies On Real-Time Dereverberation Targets](https://arxiv.org/pdf/2206.07917.pdf)
    - [End-to-End Voice Conversion with Information Perturbation](https://arxiv.org/pdf/2206.07569.pdf)
    - [Multi-instrument Music Synthesis with Spectrogram Diffusion](https://arxiv.org/pdf/2206.05408.pdf)

**Comment by ghlee0304** [comment]
###Arxiv (Audio and Speech Processing)
- [Automatic Prosody Annotation with Pre-Trained Text-Speech Model](https://arxiv.org/pdf/2206.07956.pdf)
- [GoodBye WaveNet - A Language Model for Raw Audio with Context of 1/2 Million Samples](https://arxiv.org/pdf/2206.08297.pdf)
- 흥미 있는 연구
    - [To Dereverb Or Not to Dereverb? Perceptual Studies On Real-Time Dereverberation Targets](https://arxiv.org/pdf/2206.07917.pdf)
    - [End-to-End Voice Conversion with Information Perturbation](https://arxiv.org/pdf/2206.07569.pdf)
    - [Multi-instrument Music Synthesis with Spectrogram Diffusion](https://arxiv.org/pdf/2206.05408.pdf)

**Comment by veritas9872** [comment]
**Understanding the Generalization Benefit of Normalization Layers: Sharpness Reduction**

https://arxiv.org/abs/2206.07085

Batchnorm, Layernorm, 등 normalization layer와 weight decay를 같이 적용했을 때 neural network 학습에 어떤 영향을 미치는지에 대해 수학적인 분석 및 실험 결과에 대한 해석입니다. Normalization은 neural network weight에 대해 scale invariance를 가지게끔 하지만 weight decay는 여전히 gradient의 magnitude에 영향을 미치기 때문에 학습 안정성에 기여한다고 저자들이 주장합니다.

또한, sharpness의 개념을 보다 체계화하기 위해 unit sphere로 projection한 weight matrix의 Hessian을 기준으로 삼아 normalization으로 인한 norm magnification issue를 해결하고자 합니다.

<img width="326" alt="스크린샷 2022-06-19 오후 5 30 51" src="https://user-images.githubusercontent.com/33523965/174472684-253726e6-1a19-41bd-a70c-d2f96874f20e.png">


**Speak Like a Dog: Human to Non-human creature Voice Conversion**

Arxiv: https://arxiv.org/abs/2206.04780
GitHub: https://github.com/suzuki256/dog-dataset
인간 목소리를 비인간 음성의 특성을 띄도록하는 연구가 공개되었습니다. 애니메이션이나 판타지 영화 등에서 짐승 및 괴물의 음성을 생성하기 위해 여러 각색을 시도해야 하는데 Speak like a dog 모델을 사용하면 기존 방법에 비해 손쉽게 비인간 음성을 생성할 수 있을 것으로 생각됩니다.

논문 중에는 StarGAN을 mel spectrogram 영상에 적용하여 인간 발화와 개의 소리 사이에 변환을 진행합니다. 인간 음성과 개의 소리 사이에 소리의 길이 등 차이를 감안하는 등 task-specific issue에 대한 설명도 있습니다.

<img width="516" alt="image" src="https://user-images.githubusercontent.com/33523965/174472929-27a112fe-fe58-4821-b363-4e7a43b839e7.png">

아래 참조를 위해 생성 음성 예시를 공유해드립니다. 일본어이고 퀄리티가 매우 높지는 않지만 시장성이 매우 높은 연구분야라고 생각됩니다.

Samples: https://drive.google.com/drive/folders/1aQ5o0Ond50nbAvZsp_me4b97j8VtLYbz

**Comment by kimyoungdo0122** [comment]
- News
    - Google LaMDA에 대한 이슈
        - [LaMDA(Language Model for Dialogue Applications)](https://arxiv.org/abs/2201.08239)는 구글에서 개발한 137B 크기의 대화형 서비스에 들어가는 모델
        - 근데 LaMDA라는 대화형 모델에 인지, 지각이 있다고 Google Responsible AI의 Blake Lemoine이라는 연구자가 주장함
        - 이에 대한 [워싱턴포스트와의 인터뷰](https://www.washingtonpost.com/technology/2022/06/11/google-ai-lamda-blake-lemoine/)
        - LaMDA에 어떤 인간적인(?) 인지능력이 있다는 주장하는 이유는 [LaMDA의 대화능력](https://cajundiscordian.medium.com/is-lamda-sentient-an-interview-ea64d916d917) 때문
        - 그리고 LLM이 인지능력을 갖고 있다는 주장에 반대하는 의견
        [Andrew Ng 교수님 뉴스레터](https://read.deeplearning.ai/the-batch/issue-149/)
        [Melanie Mitchell 언론사 인터뷰](https://www.msnbc.com/opinion/msnbc-opinion/google-s-ai-impressive-it-s-not-sentient-here-s-n1296406) 
        - 요지는 AI의 인지능력에 대한 과학적인 척도가 없는 상황에서 LaMDA와 같은 LLM에 인지능력이 있다는 주장은 인정할 수 없다는 것
        - 결국 구글은 르모인을 기밀 유지를 어겼다는 내용으로 휴직처리
        - 회사가 LaMDA가 인지능력을 가졌다는 주장에 대한 검증 자체를 기피하며, 구글이 AI 윤리 이슈와 이를 다루는 구성원을 홀대한다는 핑퐁 발생..

**Comment by hollobit** [comment]
AI/ML Trustworthiness characteristics matrix  
https://github.com/hollobit/WG3_TCM  
- ISO/IEC JTC1/SC 42/WG3 roadmapping AHG에서 작업 중인 matrix 
- SC42에서 개발된(중인) 표준들과 trustworthiness characteristic들과의 연관성을 mapping 하는 분류 작업 시작 
- github을 이용한 첫번째 형태, 앞으로 어떻게 이쁘게 직관적으로 관계와 관련성들을 잘 표현할지 계속 고민할 예정

**Comment by nick-jhlee** [comment]
### News
- ICML Workshops results out! (almost)

### Papers
- [Asynchronous SGD Beats Minibatch SGD Under Arbitrary Delays](https://arxiv.org/abs/2206.07638)
- [Prioritized Training on Points that are Learnable, Worth Learning, and Not Yet Learnt](https://arxiv.org/pdf/2206.07137.pdf) (ICML 2022)
- [Wide Bayesian neural networks have a simple weight posterior: theory and accelerated sampling](https://arxiv.org/pdf/2206.07673.pdf) (ICML 2022)
- [A Deep Dive into Dataset Imbalance and Bias in Face Identification](https://arxiv.org/pdf/2203.08235.pdf)

**Comment by nick-jhlee** [comment]
<img width="699" alt="Screen Shot 2022-06-19 at 11 09 45 PM" src="https://user-images.githubusercontent.com/38721329/174485253-821e6595-8bf5-4b14-b428-2d8a62bc23e2.png">
